src/setup.ts

```diff
@@ -51,6 +53,64 @@ const labelNodes = () => {
 
 const setup = () => window.onload = () => {
     labelNodes();
+    let comments: Promise<NTRComment[]> = getComments();
+    let lastSelected: HTMLElement | undefined = undefined;
+    let lastSelectedBackground: string = "";
+
+    comments.then((com: NTRComment[]) => {
+        com.forEach(element => {
```

Comment:
Can you extract this inner lambda passed to forEach into a variable so it has a name and is accessible elsewhere? Ideally somewhere in comments.ts